python/legate_dataframe/lib/parquet.pyx

```diff
@@ -80,6 +86,13 @@ def parquet_read(files, *, columns=None) -> LogicalTable:
         otherwise must be a path or an iterable of paths (or strings).
     columns
         List of strings selecting a subset of columns to read.
+    chunk_by_rows
+        If set to ``True`` the read operation will not be chunked into row groups.
```

Comment:
This deserves perhaps a little more explanation. Also the name now looks a little weird to me in that chunk_by_rows means don't use row_groups. Maybe ignore_row_groups?

I.e. Parquet files may arrive with defined row_groups - by default we use these row groups to enable parallelism, where each legate worker can process one or more row groups. Sometimes a parquet arrives with very large row groups (in the order of 10GB or more). This limits us, particularly on the CPU, if we can only launch 1 thread per 10GB of data. Setting this to true ignores metadata defined row_groups and creates a custom partitioning.

Reply:
Let's go with `ignore_row_groups`, I don't like either name much TBH (also it's difficult to actually say what the trade-offs/resource use is exactly).

I.e. on the CPU, we are still limited with this... since arrow seems to have no way to skip rows quickly (e.g. without full decompression).

I am wondering if there is a predicate API that can do this (I think there is a predicate API, but I am not sure if it has a row number logic).